src/FixedDataTableRow.js

```diff
@@ -452,16 +476,47 @@ class FixedDataTableRow extends React.Component {
     this._initialRender = false;
   }
 
+  shouldComponentUpdate(nextProps) {
+    // if row is still fake or still not visible then no need to update
+    if (
+      !this.props.visible && !nextProps.visible ||
+      this.props.fake && nextProps.fake
+    ) {
+      return false;
+    }
+
+    // if row's visibility or fakeness has changed, then update it
+    if (
```

Comment:
I just mean if we do this if statement before the if statement above, then the if statement above doesn't need to check both props & nextProps since we'll know both props & nextProps will be the same for the visible & fake fields

Reply:
aha, that's smart. Will do, thanks.